build/dotnet/dotnet-install.ps1

```diff
@@ -115,25 +119,73 @@ function Get-Version-Info-From-Version-Text([string]$VersionText) {
     return $VersionInfo
 }
 
+function Load-Assembly([string] $Assembly) {
+    try {
+        Add-Type -Assembly $Assembly | Out-Null
+    }
+    catch {
+        # On Nano Server, Powershell Core Edition is used.  Add-Type is unable to resolve base class assemblies because they are not GAC'd.
+        # Loading the base class assemblies is not unnecessary as the types will automatically get resolved.
```

Comment:
Idk. This file is just a copy of https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/obtain/dotnet-install.ps1